sdk/highlight-java/highlight-common/src/main/java/io/highlight/sdk/common/HighlightOptions.java

```diff
@@ -9,7 +9,7 @@
  * Represents the options for highlighting in a project, including project ID,
  * backend URL, environment, version, and default attributes.
  */
-public record HighlightOptions(String projectId, String backendUrl, String enviroment, String version, boolean metric,
+public record HighlightOptions(String projectId, String backendUrl, String enviroment, String version, String serviceName, boolean metric,
```

Comment:
could you add a `serviceVersion` as well per the [docs](https://www.highlight.io/docs/general/product-features/general-features/services)?